### PR TITLE
Interactive raster analytics using Jupyter Dashboards updated sample notebook

### PR DESCRIPTION
**Pull request by shivanip32**
This PR has updated sample notebook for  "Interactive raster analytics using Jupyter Dashboards"

-----

# Checklist

Please go through each entry in the below checklist and mark an 'X' if that condition has been met. Every entry should be marked with an 'X' to be get the Pull Request approved.

- [x] All `import`s are in the first cell? First block of imports are standard libraries, second block are 3rd party libraries, third block are all `arcgis` imports? Note that in some cases, for samples, it is a good idea to keep the imports next to where they are used, particularly for uncommonly used features that we want to highlight.
- [x] All `GIS` object instantiations are one of the following?
    - `gis = GIS()`
    - `gis = GIS('https://www.arcgis.com', 'arcgis_python', 'P@ssword123')`
    - `gis = GIS(profile="your_online_profile")`
    - `gis = GIS('https://pythonapi.playground.esri.com/portal', 'arcgis_python', 'amazing_arcgis_123')`
    - `gis = GIS(profile="your_enterprise_portal")`
- [ ] If this notebook requires setup or teardown, did you add the appropriate code to `./misc/setup.py` and/or `./misc/teardown.py`?
- [x] If this notebook references any portal items that need to be staged on AGOL/Python API playground, did you coordinate with a Python API team member to stage the item the correct way with the api\_data\_owner user?
- [x] Code refactored & split out across multiple cells, useful comments?
- [ ] Consistent voice/tense/narrative style? Thoroughly checked for typos?
- [ ] All images used like `` instead of ``? All map widgets contain a static image preview? (Call `mapview_inst.take_screenshot()` to do so)
- [ ] All file paths are constructed in an OS-agnostic fashion with `os.path.join()`? (Instead of `r"\foo\bar"`, `os.path.join(os.path.sep, "foo", "bar")`, etc.)
- [ ] **IF YOU WANT THIS SAMPLE TO BE DISPLAYED ON THE DEVELOPERS.ARCGIS.COM WEBSITE**, ping @ DavidJVitale so he can add it to the list for the next deploy

### CODE REVIEW

File: samples/02_power_users_developers/jupyter_dashboard_for_raster_analytics.ipynb [hunk 1]
```diff
@@ -130,7 +130,7 @@
     "\n",
```
Comment (priyankatuteja) [inline samples/02_power_users_developers/jupyter_dashboard_for_raster_analytics.ipynb:130]: <p>To make the search more robust, i think it's a good idea to include tags:<a href="https://www.arcgis.com/home/search.html?t=content&amp;q=tags:%22Landsat%20on%20AWS%22" target="_blank">Landsat on AWS</a>,&nbsp;<a href="https://www.arcgis.com/home/search.html?t=content&amp;q=tags:%22Landsat%22" target="_blank">Landsat</a>,&nbsp;<a href="https://www.arcgis.com/home/search.html?t=content&amp;q=tags:%22Multispectral%22" target="_blank">Multispectral</a>,&nbsp;<a href="https://www.arcgis.com/home/search.html?t=content&amp;q=tags:%22Multitemporal%22" target="_blank">Multitemporal</a>,&nbsp;<a href="https://www.arcgis.com/home/search.html?t=content&amp;q=tags:%22imagery%22" target="_blank">imagery</a>,&nbsp;<a href="https://www.arcgis.com/home/search.html?t=content&amp;q=tags:%22landsat%208%22" target="_blank">landsat 8</a>,&nbsp;<a href="https://www.arcgis.com/home/search.html?t=content&amp;q=tags:%22temporal%22" target="_blank">temporal</a>,&nbsp;<a href="https://www.arcgis.com/home/search.html?t=content&amp;q=tags:%22MS%22" target="_blank">MS</a> in the search. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/769/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

**Review by priyankatuteja — CHANGES_REQUESTED** [review]
I have suggested a minor addition in the search

**Review by AtmaMani — APPROVED** [review]

**Comment by AtmaMani** [comment]
@priyankatuteja could you approve this PR if it looks good to you

**Review by priyankatuteja — APPROVED** [review]
Thanks @shivanip32

**Pull request merged**